sktime/transformations/series/tests/test_window_summarizer.py

```diff
@@ -76,6 +76,14 @@ def count_gt100(x):
         count_gt100: [[3, 2]],
     }
 }
+
+kwargs_series = {
+    "lag_feature": {
+        "mean": [[-1, 1], [-1, 2]],
+    }
+}
+y_series = pd.DataFrame(np.arange(10))
```

Comment:
You are overwriting y_series. You could probably reuse `y_pd` which is defined previously for the test.